extensions/ComponentWatching/Extension.pm

```diff
@@ -467,15 +467,18 @@ sub bugmail_relationships {
 #
 
 sub _getWatches {
-    my ($user) = @_;
+    my ($user, $watch_id) = @_;
     my $dbh = Bugzilla->dbh;
 
+    $watch_id = ($watch_id =~ /^(\d+)$/) ? $1 : undef;
```

Comment:
Also missed this error in the log:

Use of uninitialized value $watch_id in pattern match (m//) at /app/extensions/ComponentWatching/Extension.pm line 473.

Change to:

$watch_id = (defined $watch_id && $watch_id =~ /^(\d+)$/) ? $1 : undef;

Reply:
@dklawren Thanks! Will fix them and finish the frontend stuff later this week.